tools/rsdl/rapid-cli/rapid.rdm.transformation.tests/data/type-facets/sample.csdl.xml

```diff
@@ -1,20 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
-<edmx:Edmx Version="4.0"
-  xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
   <edmx:DataServices>
-    <Schema Namespace="Model"
-      xmlns="http://docs.oasis-open.org/odata/ns/edm">
+    <Schema Namespace="Model" xmlns="http://docs.oasis-open.org/odata/ns/edm">
       <ComplexType Name="A" OpenType="true">
         <Property Name="prop00" Type="Edm.String" Nullable="false" />
         <Property Name="prop01" Type="Edm.String" Nullable="false" MaxLength="9" />
         <Property Name="prop02" Type="Edm.String" MaxLength="9" />
         <Property Name="prop03" Type="Collection(Edm.String)" Nullable="false" MaxLength="9" />
-        <Property Name="prop04" Type="Collection(Edm.String)" MaxLength="9" />
+        <Property Name="prop04" Type="Collection(Edm.String)" Nullable="true" MaxLength="9" />
         <Property Name="prop05" Type="Edm.Decimal" Nullable="false" Scale="variable" />
         <Property Name="prop06" Type="Edm.Decimal" Nullable="false" Precision="9" Scale="2" />
         <Property Name="prop07" Type="Edm.Decimal" Precision="9" Scale="2" />
         <Property Name="prop08" Type="Collection(Edm.Decimal)" Nullable="false" Precision="9" Scale="2" />
-        <Property Name="prop09" Type="Collection(Edm.Decimal)" Precision="9" Scale="2" />
+        <Property Name="prop09" Type="Collection(Edm.Decimal)" Nullable="true" Precision="9" Scale="2" />
```

Comment:
@chrisspre Nullable needs to be explicitly stated in V4 collections

Reply:
That would then be a bug in [Microsoft.OData.Edm](https://www.nuget.org/packages/Microsoft.OData.Edm/)
@ralfhandl is [this the section](https://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html#sec_Nullable) you are referring to ?
quote "_in OData 4.01 responses a collection-valued property MUST specify a value for the Nullable attribute_."

Reply:
Yes, and [its predecessor in V4](http://docs.oasis-open.org/odata/odata/v4.0/errata03/os/complete/part3-csdl/odata-v4.0-errata03-os-part3-csdl-complete.html#_Toc453752529):

> The absence of the Nullable attribute means it is unknown whether the collection can contain null values.